HW12/test-hw12.json

```diff
@@ -0,0 +1,19 @@
+{
+  "Id": "Policy1537365823345",
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Sid": "Stmt1537365816157",
+      "Action": [
+        "s3:CreateBucket"
+      ],
+      "Effect": "Deny",
+      "Resource": [
+        "arn:aws:s3:::test1",
+        "arn:aws:s3:::backup-test*"
+ "arn:aws:s3:::hw12-bucket"
```

Comment:
should be tabs here

Reply:
Done!
![image](https://user-images.githubusercontent.com/44346029/113304533-cd677e80-930a-11eb-8ceb-6ffe8dd15351.png)